### DISCUSSION TITLE
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'k0_.break' in 'field list') error occurs after kimai 2.23.0 version update.

### DISCUSSION THREAD

**Discussion by seleondevel** [kimai/kimai#5101] [Plugins]
![스크린샷 2024-10-09 164338](https://github.com/user-attachments/assets/d4e31871-bd12-49d4-bf49-f5c9577f432f)

![스크린샷 2024-10-09 165856](https://github.com/user-attachments/assets/11c45f20-ce12-4f28-ba40-c22a60b29774)

SQLSTATE[42S22]: Column not found: 1054 Unknown column 'k0_.break' in 'field list') error occurs after kimai 2.23.0 version update.

What should I do?

I was going to reflect the Kimai update on the company live server tomorrow, but I'm baffled by an unexpected error.

**Comment by kevinpapst** [comment]
https://www.kimai.org/documentation/updates.html

**Reply by seleondevel** [reply]
Sorry, I accidentally missed the file below in the /migrations folder while working on merge to git. I updated it normally.
Version20240920105524.php
Version20240926111739.php
